triton-client/src/components/ProjectDiskUsage/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { FunctionComponent, useEffect } from "react"
+import { selectConstants } from "@store/constants"
+import { useAppDispatch, useAppSelector } from "@store/hooks"
+import { fetchConstants } from "@store/thunks"
+import { Col, Flex, Layout, Progress, Row } from "antd"
+import { SUPPORTED_DOWNLOAD_TYPES } from "../../constants"
+import { dataSize } from "../../functions"
+import { ProjectDiskUsageProps } from "./interfaces"
+import { Content } from "antd/es/layout/layout"
+import { Typography } from "antd"
+import "./index.scss"
+const { Text, Title } = Typography
+
+const ProjectDiskUsage: FunctionComponent<ProjectDiskUsageProps> = ({
+    projectExternalId,
+}) => {
+    const dispatch = useAppDispatch()
+    const constants = useAppSelector(selectConstants)
+
+    useEffect(() => {
+        dispatch(fetchConstants())
+    }, [dispatch])
+
+    const diskUsage = useAppSelector(
+        (state: any) =>
+            state.projectsState.projectsById[projectExternalId]?.diskUsage,
+    )
+    return (
+        <>
+            {SUPPORTED_DOWNLOAD_TYPES.map((type) => {
+                const usage = diskUsage ? diskUsage[type] : 0
+                const capacity = constants.diskCapacity[type]
+                const [scaledUsage, usageUnit] = diskUsage
+                    ? dataSize(diskUsage[type])
+                    : []
+                const [scaledCapacity, capacityUnit] = dataSize(
+                    constants.diskCapacity[type],
+                )
+                return (
+                    <Layout>
+                        <Content className="disk-type-usage-container">
+                            <Title level={5}>{type}</Title>
+                            <Text>
+                                {`${scaledUsage} ${usageUnit} / ${scaledCapacity} ${capacityUnit}`}
+                            </Text>
+
+                            <Flex gap="small" vertical>
+                                <Progress
+                                    percentPosition={{
+                                        align: "center",
+                                        type: "inner",
+                                    }}
+                                    size={["100%", 20]}
+                                    percent={(usage / capacity) * 100}
```

Comment:
Percentage is too precise.
![image](https://github.com/user-attachments/assets/d1ea5559-69fc-4cc3-8aac-f3e41ba5b002)

Reply:
Also Andres raised a concern that our progress bars kinda looks like download progress. I think we should change the way we present them. Maybe it's better to use circular progressbar with disk usage/capacity in the middle?

Reply:
It also looks a bit cramped when percentage is at 0%

Reply:
> Also Andres raised a concern that our progress bars kinda looks like download progress. I think we should change the way we present them. Maybe it's better to use circular progressbar with disk usage/capacity in the middle? ![image](https://private-user-images.githubusercontent.com/15523177/349184760-d28b3c35-d695-4651-acb1-0a9d9cbb3983.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjExNDU3MDcsIm5iZiI6MTcyMTE0NTQwNywicGF0aCI6Ii8xNTUyMzE3Ny8zNDkxODQ3NjAtZDI4YjNjMzUtZDY5NS00NjUxLWFjYjEtMGE5ZDljYmIzOTgzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE2VDE1NTY0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY4YTg2ODNiOThlNDNhZDM4MzVjZjVmYjQxNTQzYjMyOWY1ZTcwZmZmODMxMmJmNGY5YWNlZGZhOWZkOGIzYWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZKKDpFNOmRX17UIJIpf9IQRsuPQCY4x0-c0HaOU_Rhs)

Another option could be to just remove percentage and instead change the background so that the max bar length is visible.